packages/openmrs/src/Utils.js

```diff
@@ -15,28 +15,39 @@ export const prepareNextState = (state, response, callback) => {
   return callback(nextState);
 };
 
-export async function request(state, method, path, data, params) {
-  const { instanceUrl, username, password } = state.configuration;
-  const headers = makeBasicAuthHeader(username, password);
+export async function request(state, method, path, options = {}) {
 
-  const options = {
+  const {
+    baseUrl = '',
```

Comment:
If we're going to default the baseURL, we should default it to something like the main openMRS platform (assuming there is such a thing).

Alternatively, and probably better, is we throw an error if baseURL is not set. Because we shouldn't allow users to call out to non openMRS systems from this adaptor

Reply:
@josephjclark  we have two different urls and these can change anytime. Should we check for both of them?

Reply:
Can you say more about this? I don't think I understand

Reply:
@josephjclark  in openmrs, one can give either `https://demo.openmrs.org/openmrs`, or `http://msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com/openmrs`.